pom.xml

```diff
@@ -39,11 +39,11 @@
     <!-- Execution environment supported by Cloud Tools for Eclipse -->
     <requiredExecutionEnvironment>JavaSE-1.8</requiredExecutionEnvironment>
 
-    <tycho.version>1.5.0</tycho.version>
-    <tycho-extras.version>1.5.0</tycho-extras.version>
+    <tycho.version>2.5.0</tycho.version>
+    <tycho-extras.version>2.5.0</tycho-extras.version>
     <product.version.qualifier.suffix/>  <!-- 0-length string by default -->
     <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
-    <eclipse.target>2018-09</eclipse.target> <!-- the default build -->
+    <!-- <eclipse.target>latest-release</eclipse.target> --> <!-- the default build -->
```

Comment:
I guess your intention is to have this as documentation, which is nice. But what usually happens is that someone who sees this later just cleans up these kinds of comments, thinking that they are unnecessary code or mistakenly left.